packages/builder-core/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./build",
+    "rootDirs": ["./src"]
```

Comment:
why do we need to specify `rootDirs`? I think `baseUrl` is preferred.

Reply:
changing to baseUrl: src

Reply:
Removed. Root project config is extended.